set.mm

```diff
@@ -17442,7 +17454,7 @@ arbitrary numbers of universal quantifiers in the consequent (the series
      (membership) predicate.  This is read " ` x ` is an element of
      ` y ` ," " ` x ` is a member of ` y ` ," " ` x ` belongs to ` y ` ,"
      or " ` y ` contains ` x ` ."  Note:  The phrase " ` y ` includes
-     ` x ` " means " ` x ` is a subset of ` y ` ;" to use it also for
+     ` x ` " means " ` x ` is a subset of ` y ` "; to use it also for
```

Comment:
So you apply the "logical quotation style" already? For me this is OK.

Reply:
I did the occurrences of `;"` (very few of them). The other ones are `."` and `,"` (there are no other punctuation signs immediately preceding a double quote), but there are many of them, so it's for a future PR... Also, I haven't looked at single quotes.

Reply:
@avekens : two questions:
1. do you plan to ask Mario or David to have the permission to merge (and officially approve PR, and edit posts) ? This may come in handy from time to time if other contributors are away for some reason.
2. have you received my email from last week ? (sent at yahoo.de)